examples/xeval_test.ts

```diff
@@ -62,7 +62,8 @@ Deno.test({
   },
 });
 
-Deno.test("xevalCliSyntaxError", async function () {
+// TODO(kt3k): Enable this test
+Deno.test("xevalCliSyntaxError", { ignore: true }, async function () {
```

Comment:
This is also disabled temporarily (because it's failing with canary). I created issue for it https://github.com/denoland/deno_std/issues/3343